src/device.rs

```diff
@@ -299,6 +299,9 @@ pub unsafe extern "C" fn wgpuDeviceCreateBindGroup(
         layout: descriptor.layout,
         entries: Cow::Borrowed(&entries),
     };
+    if entries.len() == 0 {
+        panic!("Bind groups with zero entries not allowed.");
+    }
```

Comment:
Otherwise, without validation layers, I suppose you'd not see an error but simply crash.

Reply:
But its a bit ahead of a decision on https://github.com/gfx-rs/wgpu/issues/240

Reply:
this shouldn't be handled in wgpu-native

Reply:
Agreed. Will remove it.

Do you think the changes to the triangle example should still be applied, or do you prefer closing this PR, since it looks like the problem will be resolved?

Reply:
I think it's good to still fix the example. Empty bind groups aren't very useful :)

Reply:
Ok,. I removed the check.